packages/mdc-auto-init/index.js

```diff
@@ -41,14 +42,18 @@ export default function mdcAutoInit(root = document, warn = CONSOLE_WARN) {
     }
 
     // TODO: Should we make an eslint rule for an attachTo() static method?
-    const component = Ctor.attachTo(node);
+    component = Ctor.attachTo(node);
     Object.defineProperty(node, ctorName, {
       value: component,
       writable: false,
       enumerable: false,
       configurable: true,
     });
   }
+
+  if (component !== null) {
```

Comment:
There might be multiple components initialize with `mdc-auto-init` method and my original thought is each of them should have their own `init:end` event. 

If you are interested in just having one custom event about the whole auto-init end, maybe we shouldn't do that from a component level.